tests/main_pipeline_alevin.test

```diff
@@ -0,0 +1,63 @@
+nextflow_pipeline {
+
+    name "Test Workflow main.nf"
+    script "main.nf"
+
+    test("test-dataset_alevin_aligner") {
+
+        when {
+            // the rest is taken from shared config
+            params {
+                aligner    = 'alevin'
+                outdir     = "${outputDir}/results_alevin"
+
+                // Limit resources so that this can run on GitHub Actions -- for some reason it had not been taken from shared config
+                max_cpus   = 2
+                max_memory = '6.GB'
+                max_time   = '6.h'
+            }
+        }
+
+        then {
+
+            //
+            // General assertions
+            //
+
+            // Did it finish successfully?
+            assert workflow.success
```

Comment:
recommend wrapping all assertions in `assertAll()` block....rest LGTM

Reply:
Done.